packages/functoria/functoria.2.1.0/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "1.2"
+maintainer:   "Gabriel Radanne <drupyog@zoho.com>"
+authors:      [ "Thomas Gazagnaire"
+                "Anil Madhavapeddy"
+                "Dave Scott"
+                "Thomas Leonard"
+                "Gabriel Radanne" ]
+homepage:     "https://github.com/mirage/functoria"
+bug-reports:  "https://github.com/mirage/functoria/issues"
+dev-repo:     "https://github.com/mirage/functoria.git"
+doc:          "https://mirage.github.io/functoria/"
+license:      "ISC"
+tags:         ["org:mirage"]
+
+build: [
+  ["jbuilder" "subst"] {pinned}
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+build-test: [
+  ["jbuilder" "runtest"]
+]
+depends: [
+  "jbuilder" {build & >="1.0+beta10"}
+  "base-unix"
+  "cmdliner" {>= "0.9.8"}
+  "rresult"
+  "astring"
+  "fmt"
+  "ocamlgraph"
+  "logs"
+  "bos"
+  "fpath"
+  "ounit" {test}
```

Comment:
@avsm This may be caught by the CI but it may be shorter to tell you immediately, could you please adjust this so that we can close https://github.com/ocaml/opam-repository/issues/7868 at the same time.